CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 5.3.2
```

Comment:
This file is missing a lot of other changes, but I figured that adding them all in was outside the scope of this PR.